react-basic-practice/src/components/CustomerTable/index.tsx

```diff
@@ -1,17 +1,58 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
 import {
   Th,
   Tr,
   Thead,
   Tbody,
   Td,
   Table as ChakraTable,
+  IconButton,
 } from "@chakra-ui/react";
-import { SortingIcon } from "@icons";
-import { ICustomerTableProps } from "@types";
+import { ICustomer, ICustomerTableProps } from "@types";
 import TableRow from "../TableRow";
+import { SortingIcon } from "@icons";
+import { fetchUsers } from "@services";
+import { TypeSortDirection } from "@types";
+
+const CustomerTable: React.FC<ICustomerTableProps> = ({ columns, action }) => {
+  const [data, setData] = useState<ICustomer[]>([]);
+  const [sortConfig, setSortConfig] = useState<{
+    key: keyof ICustomer;
+    direction: TypeSortDirection;
+  } | null>(null);
```

Comment:
it should be empty `()`. We can use undefined instead of `null`

Reply:
i done